test/resources/microshift-host.resource

```diff
@@ -157,3 +158,13 @@ Command Should Fail
     ...    return_stderr=True
     ...    return_stdout=True
     Should Not Be Equal As Integers    0    ${rc}
+
+Local Command Should Work
```

Comment:
I don't think this keyword belong in this file.
This file contains keywords that making changes on microshift host, but new keyword is hypervisor/test runner specific

Reply:
Move keyword in https://github.com/openshift/microshift/pull/4662/commits/8ca4fae49a3eface9730e59abc8b33477c48d8a3 commit